web-ui/src/main/resources/catalog/js/edit/NewMetadataController.js

```diff
@@ -159,15 +172,154 @@
       }
 
       $scope.createNewMetadata = function(isPublic) {
+        var metadataUuid = "";
+
+        // If no auto-generated metadata identifier, get the value
+        if (!$scope.generateUuid && ($scope.mdIdentifierSelectedTemplateId != 1)) {
+
+          // Custom identifier
+          if ($scope.mdIdentifierSelectedTemplateId == 0) {
+            metadataUuid = $scope.urnCustom;
+
+          // Template identifier
+          } else {
+            metadataUuid = getSelectedMdIdentifierTemplate().template;
+
+            for (key in $scope.mdIdentifierTemplateTokens) {
+              var labelKey = $scope.mdIdentifierTemplateTokens[key].label;
+              metadataUuid = metadataUuid.replace("{"+labelKey+"}", $scope.mdIdentifierTemplateTokens[key].value);
```

Comment:
Did you run mvn clean install -Pjslint in the web-ui module. This line looks a bit too long ?